### ISSUE TITLE
The "Fork me on GitHub" link goes to wrong repo

### ISSUE THREAD

**Issue by styfle** [Microsoft/TypeScript-Handbook#571]
I clicked this and it took me to [Microsoft/Typescript](https://github.com/Microsoft/TypeScript) instead of the expected [Microsoft/Typescript-Handbook](https://github.com/Microsoft/TypeScript-Handbook)

![image](https://cloud.githubusercontent.com/assets/229881/25635572/99f8502c-2f4c-11e7-8c01-736045d9b2e7.png)

This is a bug, right?

**Comment by DanielRosenwasser** [comment]
No, the site in general has this as a banner. Maybe we shouldn't display it other than on the front-page. Thoughts @bowdenk7?

**Comment by styfle** [comment]
@DanielRosenwasser Even if the ribbon was changed to only show on the home page, there should still be a link from the handbook page to the this handbook repo.

Otherwise, no one is going to submit PRs because they won't know how to edit.

**Comment by perlun** [comment]
> Otherwise, no one is going to submit PRs because they won't know how to edit.

Totally agree. I think we should even go one step further and do it like [Aurelia](http://aurelia.io/hub.html#/doc/article/aurelia/framework/latest/contact-manager-tutorial/1) and [Ember](https://guides.emberjs.com/v2.13.0/) does it. That is: provide an Edit button of some form that goes _straight to the right page_ in the documentation repo (ergo this one). Check the links to try it out in action.

I find that kind of button immensely useful, and it definitely lowers the bar for people to contribute, especially when doing minuscule changes like #587.

**Comment by bowdenk7** [comment]
That's a great idea. If we open sourced the website would either of yall be interested in helping us implement this?

**Comment by styfle** [comment]
@bowdenk7 Yes, I think it would be trivial to implement.

**Comment by perlun** [comment]
@bowdenk7 Absolutely. I wasnt' even aware that the website wasn't open sourced yet. 😉
